burnman/solidsolution.py

```diff
@@ -27,7 +27,7 @@ class SolidSolution(Mineral):
     and P derivatives in J/K/mol and m^3/mol.
     """
 
-    def __init__(self, endmembers, solution_model=SolutionModel()):
+    def __init__(self, endmembers, solution_model=SolutionModel(), molar_fraction=None):
```

Comment:
This seems like a bug.  In some places you are calling it molar_fraction, and in others you are calling it molar_fractions